pmix.sty

```diff
@@ -483,6 +483,30 @@
 \newcommand{\refAttributeItem}[1]{\index[index_attribute]{#1}\hyperref[attr:#1]{\code{#1}}}
 \newcommand{\refattr}[1]{\refAttributeItem{#1}}
 
+\newenvironment{signature}[1]{
```

Comment:
The commit message for this should drop the last line about shorthand commands for headers